ArcBicepTemplate/FullAzureMonitorMetricsProfile.bicep

```diff
@@ -1,48 +1,7 @@
 param azureMonitorWorkspaceResourceId string
 
 @allowed([
-  'eastus2euap',
-  'northcentralus',
-  'eastus',
-  'australiacentral',
-  'australiasoutheast',
-  'brazilsouth',
-  'canadacentral',
-  'centralindia',
-  'centralus',
-  'eastasia',
-  'eastus2',
-  'northeurope',
-  'norwayeast',
-  'southafricanorth',
-  'southcentralus',
-  'southeastasia',
-  'uaenorth',
-  'uksouth',
-  'westcentralus',
-  'westeurope',
-  'westus',
-  'westus2',
-  'francesouth',
-  'germanywestcentral',
-  'israelcentral',
-  'italynorth',
-  'japanwest',
-  'koreacentral',
-  'switzerlandnorth',
-  'brazilsoutheast',
-  'francecentral',
-  'ukwest',
-  'koreasouth',
-  'switzerlandwest',
-  'japaneast',
-  'swedencentral',
-  'canadaeast',
-  'norwaywest',
-  'southindia',
-  'australiaeast',
-  'swedensouth',
-  'usgovvirginia'
+  'eastus2euap', 'northcentralus', 'eastus', 'australiacentral', 'australiasoutheast', 'brazilsouth', 'canadacentral', 'centralindia', 'centralus', 'eastasia', 'eastus2', 'northeurope', 'norwayeast', 'southafricanorth', 'southcentralus', 'southeastasia', 'uaenorth', 'uksouth', 'westcentralus', 'westeurope', 'westus', 'westus2', 'francesouth', 'germanywestcentral', 'israelcentral', 'italynorth', 'japanwest', 'koreacentral', 'switzerlandnorth', 'brazilsoutheast', 'francecentral', 'ukwest', 'koreasouth', 'switzerlandwest', 'japaneast', 'swedencentral', 'canadaeast', 'norwaywest', 'southindia', 'australiaeast', 'swedensouth', 'usgovvirginia'
```

Comment:
Could you please modify all templates (ARM, Bicep, Policy, Terraform, etc.) to eliminate the allowed region list entirely? This change would streamline our processes by removing the need for constant updates whenever new regions are added.

Even without a predefined list, if a customer selects an unsupported region, they'll receive an error message from the Resource Provider indicating that the region isn't supported. This should help with maintenance overhead of these files.

Reply:
Thanks Kaveesh! It makes sense, let me update this.

Reply:
I have updated the templates now. Please take a look.